src/docs/exiting-developer-mode.md

```diff
@@ -1,30 +1,31 @@
-# Exiting Developer Mode
+# Developer Mode
```

Comment:
This should specify what about developer mode we're going to be talking about. line 28 still titles "Entering", this should be titles "exiting"

Reply:
exiting-developer-mode.md is not sourced in the docs. I removed it when I migrated the info in here to the FAQ. I don't know if it's still there after the edits to the FAQ.

Reply:
The page can be found when using the search or by its URL, but its not referenced anywhere else. Feel free to remove it via a new PR